src/app/landing/LandingPage.jsx

```diff
@@ -99,10 +109,15 @@ class LandingPage extends Component {
             </div>
 
             <picture>
+              <source
+                className="OnboardingImage"
+                media="(min-aspect-ratio: 2/3)"
+                srcSet="img/landing_page/step3.png"
```

Comment:
Woah, how did the file size for this step go up by 950%?
Unless GitHub is just confused, it claims this and step2, on the desktop versions, are now over 1MB in size. Is that intentional?

Reply:
Following up with Di to reduce image sizes